source/_docs/mqtt/service.markdown

```diff
@@ -48,6 +48,13 @@ qos: 2
 retain: true
 ```
 
+Example of how to publish a Tasmota command (LedPower 2):
+
+```yaml
+topic: your_tasmota_device_name_topic/cmnd/LEDPOWER
+payload: 2
+```
```

Comment:
This is a duplicate of the very first example (with different values).

Why add duplicates?